src/services/serverPush.ts

```diff
@@ -14,6 +14,7 @@ interface ISong {
   trimStartSeconds: (number | null);
   trimEndSeconds: (number | null);
   timesPlayed: number;
+  playNext?: boolean;
```

Comment:
This should be required field. In the future it would be awesome if we could share type definitions between frontend and backend. This is one of those places where TypeScript shines 😄